indigo/analysis/terms/base.py

```diff
@@ -51,14 +51,12 @@ def find_terms(self, doc):
         self.find_term_references(doc, terms)
         self.renumber_terms(doc)
 
-    def fix_terms_in_document(self, document):
-        """ Do everything other than guessing at defined terms in a Document.
-            Run this before saving an updated document when a portion was edited separately.
+    def link_terms_in_provision(self, document):
```

Comment:
Minor: this method name doesn't make senses in this context, because there's no provision information. `link_terms_in_document` or `find_term_usages_in_document` or something would make more sense to me.